src/Command/ResetEndUserRequestCountCommand.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace App\Command;
+
+use App\Entity\EndUser;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+#[AsCommand(
+    name: 'app:reset-end-user-request-count',
+    description: 'Reset requestCountForToday to 0 for all EndUser entities',
+)]
+class ResetEndUserRequestCountCommand extends Command
+{
+    private const BATCH_SIZE = 100;
+
+    public function __construct(
+        private EntityManagerInterface $entityManager
+    ) {
+        parent::__construct();
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $io = new SymfonyStyle($input, $output);
+
+        $repository = $this->entityManager->getRepository(EndUser::class);
+        $query = $repository->createQueryBuilder('e')
+            ->where('e.requestCountForToday != 0')
+            ->getQuery();
```

Comment:
instead of using doctrine query builder, directly make a SQL request that performs the UPDATE